project.clj

```diff
@@ -13,26 +13,26 @@
   :license {:name "BSD Standard License"
             :url "http://www.iplantcollaborative.org/sites/default/files/iPLANT-LICENSE.txt"}
   :manifest {"Git-Ref" ~(git-ref)}
-  :dependencies [[org.clojure/clojure "1.10.0"]
+  :dependencies [[org.clojure/clojure "1.10.2"]
                  [net.sourceforge.owlapi/owlapi-api "3.5.0"]
                  [net.sourceforge.owlapi/owlapi-apibinding "3.4.10"]
                  [net.sourceforge.owlapi/owlapi-reasoner "3.3"]
                  [me.raynes/fs "1.4.6"]
-                 [cheshire "5.8.1"]
-                 [org.clojure/data.csv "0.1.4"]
```

Comment:
This is a major version upgrade, I guess we should double-check the interface isn't changed (doesn't look like the rest of the PR touches any CSV stuff)

Reply:
I checked the release notes before I updated `project.clj`. Here's a link to the diff for good measure.

Reply:
Cool. It does look like it's basically documentation changes and designating it 1.0, so, seems good. Thanks!